source/OptimaJet.DWKit.StarterApplication/Services/HttpCurrentUserContext.cs

```diff
@@ -30,18 +30,24 @@ public class HttpCurrentUserContext : ICurrentUserContext
             this.TokenService = tokenService;
         }
 
-        private string AuthType {
-            get {
-                if (authType == null) {
+        private string AuthType
```

Comment:
I found out ctrl+k, ctrl+d auto-formats in Visual Studio Community Edition
(I switched to VS Community Edition, for debugging, as I haven't figured out a good way to handle that in vim yet)

Reply:
so, I need to figure out a good way we can get more automated styling of the C# code